client/landing/stepper/declarative-flow/internals/steps-repository/components/accent-color-control/index.tsx

```diff
@@ -111,13 +112,23 @@ const AccentColorControl = ( { accentColor, setAccentColor }: AccentColorControl
 		// Hence ensure the color picker is closed after predefined color selection
 		setColorPickerOpen( false );
 
+		recordTracksEvent( 'calypso_signup_accent_color_select', {
+			color: value,
+			global_styles_gating: shouldLimitGlobalStyles,
```

Comment:
Nitpick (non-blocking): The name chosen for this property is a bit confusing to me. `global_styles_gating` sounds like it indicates whether the gating logic is running or not, but it actually indicates whether the global styles are limited/restricted/hidden or not. So I wonder if maybe something like `global_styles_limited` or `global_styles_hidden` would be clearer.

Reply:
> it indicates whether the gating logic is running or no

Never mind, just realized that this what we want to track 😅

Reply:
Yeah, the idea is to be able to compare the performance of this dropdown with or without gating.

I wonder if another prop like `is_premium_color` could be worthwhile, but we can add it separately, I guess.